vendor/github.com/openfaas/faas/gateway/requests/requests.go

```diff
@@ -51,11 +51,13 @@ type FunctionResources struct {
 
 // Function exported for system/functions endpoint
 type Function struct {
-	Name            string  `json:"name"`
-	Image           string  `json:"image"`
-	InvocationCount float64 `json:"invocationCount"` // TODO: shouldn't this be int64?
-	Replicas        uint64  `json:"replicas"`
-	EnvProcess      string  `json:"envProcess"`
+	Name              string  `json:"name"`
+	Image             string  `json:"image"`
+	InvocationCount   float64 `json:"invocationCount"` // TODO: shouldn't this be int64?
+	Replicas          uint64  `json:"replicas"`
+	EnvProcess        string  `json:"envProcess"`
+	ReplicaCount      int     `json:"replicaCount"`
+	AvailableReplicas int     `json:"availableReplicas"`
```

Comment:
This is a vendored file, so the change must be present in [github.com/openfaas/faas/gateway/requests/requests.go](https://github.com/openfaas/faas/blob/master/gateway/requests/requests.go)

Is there a corresponding uncommitted Gopkg.toml/Gopkg.lock change?

Reply:
This change should be present in `faas` repo under a new release. After @alexellis does the faas release this PR should reference the new git tag in Gokg.toml